src/pages/Dashboard/DashboardSidebar.tsx

```diff
@@ -21,6 +24,44 @@ const promoData: PromoItem[] = [
     rightElement: <Image width='80px' overflow='hidden' borderRadius='lg' src={OnRamperLogo} />,
     isExternal: true,
   },
+  {
+    title: ['promo.savers.title', { asset: 'AVAX' }],
+    body: 'promo.savers.body',
+    cta: 'promo.savers.cta',
+    image: SaversVaultTop,
+    startDate: '2023-02-20 8:00 AM',
+    endDate: '2023-02-25 8:00 AM',
+    id: 'savers-avax',
+    href: '?type=staking&provider=THORChain%20Savers&chainId=eip155%3A43114&defaultAccountId=cosmos%3Aosmosis-1%3Aosmo182emakchj2xp0llv5k87gsannvsa0cygdx307z&assetNamespace=slip44&assetReference=60&modal=overview',
```

Comment:
Note to anyone reviewing this, can remove the `defaultAccountId` as this is code smell from a previous Cosmos SDK "fix" but realistically doesn't matter, I'll run some grep fu to remove it everywhere